package.json

```diff
@@ -101,8 +101,8 @@
     "@tippyjs/react": "^4.2.6",
     "a11y-dialog": "^7.4.0",
     "draft-js": "^0.10.5",
-    "draftail": "^2.0.0-alpha.5",
-    "draftjs-filters": "^2.5.0",
+    "draftail": "^2.0.0-rc.1",
+    "draftjs-filters": "^3.0.1",
```

Comment:
The only breaking changes are browser support removals and switch to TypeScript, so there won’t be any issues with our usage.